File: GoaOnline/src/Feedback.java
public class Feedback {
    private int feedbackId;
    private int userId;
    private String message;
    private String date;
    private String response;

    // Constructor
    public Feedback(int feedbackId, int userId, String message, String date, String response) {
        this.feedbackId = feedbackId;
        this.userId = userId;
        this.message = message;
        this.date = date;
        this.response = response;
    }

    // Methods
    public void submitFeedback(int userId, String message) {
        System.out.println("Feedback submitted by user ID: " + userId);
    }

    public void viewFeedback(int feedbackId) {
        System.out.println("Viewing feedback ID: " + feedbackId);
    }

    public void responseToFeedback(int feedbackId, String response) {
        System.out.println("Responded to feedback ID " + feedbackId);
    }
}

File: python/feedback.py
class Feedback:
    def __init__(self, feedback_id, user_id, message, date, response):
        self.feedback_id = feedback_id
        self.user_id = user_id
        self.message = message
        self.date = date
        self.response = response

    def submit_feedback(self, user_id, message):
        print(f"Feedback submitted by user ID: {user_id}")

    def view_feedback(self, feedback_id):
        print(f"Viewing feedback ID: {feedback_id}")

    def response_to_feedback(self, feedback_id, response):
        print(f"Responded to feedback ID {feedback_id}")

